debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MysqlBinaryProtocolFieldReader.java

```diff
@@ -30,7 +30,7 @@
     @Override
     protected Object readTimeField(ResultSet rs, int columnIndex) throws SQLException {
         Blob b = rs.getBlob(columnIndex);
-        if (b == null) {
+        if (b == null || b.length() == 0) {
```

Comment:
Could you please add this to a separate branch and log a `WARN/INFO` message together with all column values? We still don't know if this is correct or not so it would help us to diagnose the issue if it happens and we will find out it is not null but a different special value.

Reply:
@jpechane Will do

Reply:
Moved to a separate branch and added a few log messages.  Let me know if that looks good.

Reply:
That's exactly what I was looking for, thanks!